src/app/components/work-item-new-detail/work-item-new-detail.component.ts

```diff
@@ -320,7 +320,7 @@ export class WorkItemNewDetailComponent implements OnInit, OnDestroy, AfterViewC
           };
           // init dynamic form
           if (this.workItem.relationships.baseType.data.attributes) {
-            this.dynamicFormGroup = this.workItemTypeControlService.toFormGroup(this.workItem);
+            // this.dynamicFormGroup = this.workItemTypeControlService.toFormGroup(this.workItem);
```

Comment:
This is an irrelevant change but still, let it be here. This file is likely to be removed in near future.